long_term_deployment/AgentDescription description
---
bool success
string assigned_name
